### ISSUE TITLE
arg_utils needs “use_safetensor“ args，when we use llama model，use_safetensor will always be True

### ISSUE THREAD

**Issue by tanhzongbei** [vllm-project/vllm#947]
llama.py load_weights function have "use_safetensor" args,but in model_loader.py get_model function call model.load_weights don't have use_safetensor args，when we use llama model，use_safetensor will always be True。

**Comment by gesanqiu** [comment]
Check the implement of `prepare_hf_model_weights` in `weight_utils.py`, it will fallback to '*.bin' pattern when these are not "*.safetensors" weight files and set `use_safetensor=False`.
```Python
    if len(hf_weights_files) == 0 and use_safetensor:
        logger.warning("No *.safetensors files found, "
                       "fall back to *.bin files")
        return prepare_hf_model_weights(model_name_or_path,
                                        cache_dir=cache_dir,
                                        use_safetensor=False)
```

**Comment by WoosukKwon** [comment]
Hi @tanhzongbei This was addressed by #974. Now we have `load_format` argument, which is `auto` by default. In the `auto` mode, vLLM will first look for safetensors, and then look for `*.bin` files if no safetensors is found.

@gesanqiu Thanks for the kind answer to the question!
